experiment/generate_tests.py

```diff
@@ -293,13 +293,17 @@ def generate(self):
 
         tg_config = self.__get_testgrid_config()
 
+        annotations = prow_config.setdefault('annotations', {})
+        dashboards = 'sig-release-%s-all' % k8s_version['version']
+        tab_name = '%s-%s-%s-%s' % (fields[3], fields[4], fields[5], fields[6])
+        annotations['testgrid-tab-name'] = tab_name
+        if self.job.get('releaseBlocking'):
+            dashboards += ', sig-release-%s-blocking' % k8s_version['version']
```

Comment:
(basically a repeat of all the configs I found this in)

if order matters for alerting, this should be prepended instead of appended